fastparquet/writer.py

```diff
@@ -22,6 +22,7 @@
 
 MARKER = b'PAR1'
 NaT = np.timedelta64(None).tobytes()  # require numpy version >= 1.7
+nat = np.datetime64('NaT').view('int64')
```

Comment:
If these are global constants then they could probably use more descriptive names, like `NaT_NUMPY` and `NaT_PANDAS`